tests/testthat/setup.R

```diff
@@ -0,0 +1,4 @@
+gert_testing_signature <- git_signature(
+  name <- 'Testing Jerry',
+  email <- 'test@jerry.com'
+)
```

Comment:
It would probably be easier to just configure a git user. Either in the CI config files or here (if no existing user is configured). Or write a `local_git_user()` type of helper. Thoughts?

Reply:
CRAN has in the past yelled at me for gert modifying the `~/.gitconfig`. So we could only do it per repo, not globally.